classes/phing/tasks/ext/sonar/SonarConfigurationFileParser.php

```diff
@@ -89,7 +89,7 @@ public function parse()
             throw new BuildException($message);
         }
 
-        $lines = explode("\n", $contents);
+        $lines = preg_split ("/\r?\n/", $contents);
```

Comment:
https://travis-ci.com/github/phingofficial/phing/jobs/430758078

Reply:
Added tests specifically for /r/n versus /n.

While I'm on Windows I do run the tests in a Git Bash terminal so that combination may be exposing the issue.